static/app/components/modals/sudoModal.tsx

```diff
@@ -83,25 +84,29 @@ function SudoModal({
     superuserReason,
   } = state;
 
+  const {loadOrganization} = useContext(OrganizationLoaderContext) || {};
   const router = useRouter();
   const api = useApi();
   const location = useLocation();
 
   useEffect(() => {
-    const getAuthenticators = async () => {
+    const getAuthenticators = () => {
+      if (!loadOrganization) return;
       try {
-        const fetchedAuthenticators = await api.requestPromise('/authenticators/');
-        setState(prevState => ({
-          ...prevState,
-          authenticators: fetchedAuthenticators ?? [],
-        }));
+        loadOrganization().then(async () => {
```

Comment:
i don't think we'll need this inside of a `useEffect`, i believe you can just grab the `loadOrganization.then(()` here instead.

This is because the `OrganizationContextLoader` will initialize the promise chain, then here we're referencing the promise directly, so whenever the promise resolves it will call `.then`.